app/gateway/changelog.md

```diff
@@ -4105,6 +4105,53 @@ openid-connect
 * Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
 * Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21
 
+
+## 2.8.4.8
+**Release Date** 2024/03/26
+
+### Features
+#### Configuration
+
+* TLSv1.1 and lower is now disabled by default in OpenSSL 3.x.
+* **Performance:** Bumped the default values of `nginx_http_keepalive_requests` and `upstream_keepalive_max_requests` to `10000`. 
+These changes are optimized to work better in systems with high throughput. 
+In a low-throughput setting, these new settings may have visible effects in load balancing, where it can take more requests to start 
+using all the upstreams than before.
+
+### Fixes
+#### Configuration
+
+* Fixed an issue where an external plugin (Go, Javascript, or Python) would fail to
+apply a change to the plugin config via the Admin API.
+* Set the security level of gRPC's TLS to `0` when `ssl_cipher_suite` is set to `old`.
+ 
+#### Core
+
+* Updated the file permission of `kong.logrotate` to 644.
+* Fixed the missing router section for the output of request debugging.
+* Fixed a issue where the `/metrics` endpoint would throw an error when database was down.
+* Fixed the UDP socket leak of the DNS module.
+
+#### Plugins
+
+* [LDAP Auth Advanced](/hub/kong-inc/ldap-auth-advanced/) (`ldap-auth-advanced`)
+  * Fixed some cache-related issues which caused `groups_required` to return unexpected codes after a non-200 response.
+
+* [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced/) (`rate-limiting-advanced`)
+  * Fixed an issue with `sync_rate` setting being used with the `redis` strategy. 
+  If the Redis connection is interrupted while `sync_rate = 0`, the plugin now accurately falls back to the `local` strategy.
```

Comment:
```suggestion
  *  Fixed an issue where the `sync_rate` setting, when being used with the `redis` strategy did not properly revert to the `local` strategy if the Redis connection was interrupted with `sync_rate = 0`.
```

Reply:
Rephrased it a bit more and applied that.